package.json

```diff
@@ -42,7 +42,7 @@
     "lint": "eslint lib test",
     "jest": "jest --detectOpenHandles --runInBand",
     "test": "npm run lint && npm run jest",
-    "coverage": "npm run test -- --coverage",
+    "coverage": "npm run test -- -- --coverage",
```

Comment:
why is necessary the third set of `--`?

Reply:
Passing one `--` does not work anymore. However, putting `--` twice or putting `"--"` once before the arguments does the trick.